replica/compaction_group.hh

```diff
@@ -101,6 +101,8 @@ public:
         return _group_id;
     }
 
```

Comment:
>replica: Allow compaction_group to have more than one view
>
> In order to support incremental repair, we'll allow each
replica::compaction_group to have two logical compaction groups
(or logical sstable sets), one for repaired, another for unrepaired.

We'll need three, no? repaired, undergoing repair, unrepaired.